migrations/helpers/update_legacy_filtered_positives_helper.py

```diff
@@ -199,3 +232,76 @@ def split_mongo_samples_by_version(
     }
 
     return samples_by_version
+
+
+def update_mlwh_filtered_positive_fields_batched(
```

Comment:
As mentioned, I think the existing filtered positive rule change migration should call this more performant MLWH update method. Do you reckon it worth doing this as part of these changes, or a new "refactor" task, e.g. https://github.com/sanger/crawler/issues/173

Reply:
Actually I think @KatyTaylor was creating a new task with filtered positive rule change migration changes, so I think that can be done as part of that task. I'm not sure if that task has been created yet though

Reply:
@KatyTaylor, is this still to be done?